src/main/protobuf/soc/board.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto2";
+
+package protos.soc;
+
+message Hex {
+    enum Resource {
+        BRICK = 0;
+        ORE = 1;
+        SHEEP = 2;
+        WHEAT = 3;
+        WOOD = 4;
+        DESERT = 5;
+    }
+    required string id = 1;
+    required Resource resource = 2;
+    optional int32 number = 3;
+    repeated Vertex adjacentVertices = 4;
+}
+
+message Vertex {
+    required string id = 1;
+}
+
+message Edge {
+    required Vertex v1 = 1;
+    required Vertex v2 = 2;
+}
+
+message Port {
+    enum PortType {
+        BRICK = 0;
+        ORE = 1;
+        SHEEP = 2;
+        WHEAT = 3;
+        WOOD = 4;
+        MISC = 5;
+    }
+    required PortType portType = 1;
+    required Edge edge = 2;
+}
+
+message VertexBuilding {
+    enum BuildingType {
+        SETTLEMENT = 0;
+        CITY = 1;
+    }
+    required BuildingType buildingType = 1;
+    required Vertex vertex = 2;
+    required int32 playerPos = 3;
```

Comment:
we should standardize nomenclature. `position` vs `playerPos` vs `playerPosition` vs `player`.

Reply:
There could be a use cases for both, but then we should be explicit in when and why we deviate. In my PR I think  have `other_player_position` for example (which is a flavor of `player_position`).... Also, this just reminded me, we need to determine if we're doing this `camelCase` or `snake_case`. Let's check to see if there is any standard out htere

Reply:
looks like standard is snake_case. I will rename my variables

Reply:
Also, we might want to consider a `Player` type with `position` and `name` I think it's worthwhile as we can always just have `Player player` or `Player other_player`. `repeated Player viewable_by` etc. Gives good semantic meaning